base/mp_abort.sas

```diff
@@ -24,16 +24,16 @@
 %macro mp_abort(mac=mp_abort.sas, type=, msg=, iftrue=%str(1=1)
 )/*/STORE SOURCE*/;
 
+  %global sysprocessmode sysprocessname;
```

Comment:
SAS describes SYSPROCESSMODE as "a read-only automatic macro variable" so this could be troublesome.